src/main/java/appeng/client/gui/implementations/GuiCraftingCPU.java

```diff
@@ -91,6 +94,9 @@
 	private List<IAEItemStack> visual = new ArrayList<IAEItemStack>();
 	private GuiButton cancel;
 	private int tooltip = -1;
+	private int rows = 6;
+	private int maxRows = Integer.MAX_VALUE;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the "maxRows" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1450)